budget.md

```diff
@@ -0,0 +1,84 @@
+# Project Funding
+
+Need to pay for something on behalf of the Kubernetes Project? This is the document to read. In general there are a few rough types of costs that can be covered:
+
+- Infrastructure: testing infra, video conferencing, mailing lists, domains, etc
+- Events: SIG face-to-face meetings, developer summits, etc
+- Consulting services: docs writers, security auditors, etc
+- Community Gifts: swag codes, thank you cards, etc
+
+## How to request funding
+
+The GitHub repo github.com/kubernetes/funding is used for all funding requests. The issue templates will tell you the information that must be provided.
+
+After filing an issue please expect to wait 1 week for approval and potentially another week to a month for funding from the CNCF depending on the request size.
+
+
+## FAQ
+
+**What happens behind the scenes?**
+
+If the request is approved it will then go through the CNCF Service Desk however only a small number of people from the project can directly make requests.
+
+**Who Pays for this stuff?**
+
+The Kubernetes project relies on a number of paid services for day-to-day operations; this includes Zoom for video calls, Google cloud for test infrastructure, and Netlify for website deployments. Also, the project covers one-off costs like space for developer summits to bring developers together or swag incentivize and celebrate project successes. And these bills are paid for by CNCF members via their membership fees and at times in-kind donations.
+
+```
+.github/ISSUE_TEMPLATE/one-time-200.md
+
+---
+name: One-time funding under $200USD
+title: request: one-time $NNNUSD to fund MMM
+about: a non-recurring request with easiest process
+---
+
+<!--
+Please fill out the items and provide supporting evidence for request
+-->
+
+- **Description**: 
+- **Requesting SIG**:
```

Comment:
Drive-by question: would it be helpful to have a field for  SIG Approval here as well (just like [requiring SIG leads to sign off on creating new repos](https://github.com/kubernetes/org/blob/master/.github/ISSUE_TEMPLATE/repo-create.md#approvals)) or adding a note that the request would require approval from the relevant SIG chair?

However that probably means there is very little difference between `one-time-200.md` and `one-time-200-plus.md`...

Reply:
good idea @nikhita will add to README.